filters.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+class Filters {
+    
+  constructor(postalCode, kmFromPostalCode, minPrice, maxPrice, minSqft) {
+    this.postalCode = 'v6b1s3';
```

Comment:
You shouldn't use hardcoded values here. Instead, get the values from the constructor and assign to the fields.

Reply:
Done.